frontend/src/components/pages/TeamDashboard/index.jsx

```diff
@@ -59,14 +64,20 @@ function DashboardUnconnected({
       userIds: team ? team.members.map((member) => member.userId) : [],
     }),
   };
+  if (activityLogDayCounts.length === 0)
+    getActivityLogCountsByDayForUsers({
+      activityLogDayCounts,
+      eventTypes,
+      userIds: team ? team.members.map((member) => member.userId) : [],
+    });
 
   return <Presentation {...props} />;
 }
 
 function mapStateToProps(state) {
   return {
-    teams: state.apiEntities.teams || {},
-    authUser: state.App.authUser || {},
+    teams: state.apiEntities.teams,
+    authUser: state.App.authUser,
 
     activityLogDayCounts: Object.values(
       state.apiEntities.activityLogDayCounts || {}
```

Comment:
```
> o = {}
{}
> values = Object.values(o)
[]
> values2 = Object.values(o)
[]
> values === values2
false
```

Can you please always remove things that could recalculate props 

This code would be more appropriate:
```
activityLogDayCounts: state.apiEntities.activityLogDayCounts
```